EnvironmentSetup.ps1

```diff
@@ -68,7 +68,7 @@ if ($microbuild -or ($vstarget -eq "15.0" -and -not $skipRestore)) {
     Write-Output ""
     Write-Output "Installing Nuget MicroBuild packages"
 
-    & "$rootdir\Nodejs\.nuget\nuget.exe" restore "$rootdir\Nodejs\Setup\swix\packages.config" -PackagesDirectory "$packagedir"
+    & "$rootdir\Nodejs\.nuget\nuget.exe" restore "$rootdir\Nodejs\Setup\swix\project.json" -PackagesDirectory "$packagedir" -ConfigFile "$rootdir\Nodejs\.nuget\nuget.config"
```

Comment:
I recall this config file causing some issue before. Make sure this doesn't break the local or nightly build.